src/Hive.IO/Hive.IO/Forms/ConversionTechPropertiesViewModel.cs

```diff
@@ -175,6 +175,8 @@ public ModuleTypeRecord ModuleType
             }
         }
 
+        public string Description => IsSurfaceTech ? ModuleType.Description : Defaults[Name].Description;
```

Comment:
so do I understand correctly, if it's not a surfaceTech, `description` will simply be not instantiated?

Reply:
If it's not a surface tech, then the description is taken from the `Defaults[Name]` object.

Reply:
...which is defined were for non-surface techs?

Reply:
`private static Dictionary<string, ConversionTechDefaults> Defaults`

Which is actually populated by Json.Net - reading in the resource file `Hive.IO.EnergySystems.conversion_technology_defaults.json`. See the `public struct ConversionTechDefaults` for the properties it defines. One of which is a `string Description`. We currently just don't fit anything into it. But I guess we _could_. If we knew where to put it...